authors/settings.py

```diff
@@ -22,7 +22,7 @@
 SECRET_KEY = '7pgozr2jn7zs_o%i8id6=rddie!*0f0qy3$oy$(8231i^4*@u3'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
Setting Debug to False will make it hard for the team to trace any errors that may occur when the app is running. We need to set it to True for now and disable it only for the production environment

Reply:
@jomasim I agree with you.
I consider this PR proves that it is possible to achieve the objective of customizing 404 error in the backend but at a disadvantage during the development phase. 
My research has not yielded a workable alternative.